src/options.h

```diff
@@ -72,8 +72,8 @@ struct __ScrotOptions {
 };
 
 void optionsParse(int, char**);
-char* nameThumbnail(char*);
-void optionsParseFileName(char*);
+char* nameThumbnail(const char*);
```

Comment:
As this function belongs to this file it should continue with the style, something like this:
```C
char* optionsNameThumbnail(const char*);
```

Reply:
I understand that you did not write this function, but do me a favor and change the name in your PR, thanks

Reply:
Sure!